propagation.go

```diff
@@ -178,13 +187,30 @@ func extractTraceContext(opaqueCarrier interface{}) (SpanContext, error) {
 		return spanContext, ot.ErrSpanContextNotFound
 	}
 
-	if !spanContext.Suppressed && (spanContext.SpanID == 0 != (spanContext.TraceIDHi == 0 && spanContext.TraceID == 0)) {
-		return spanContext, ot.ErrSpanContextCorrupted
+	// When the context is not suppressed and one of Instana ID headers set.
+	if !spanContext.Suppressed &&
+		(spanContext.SpanID == 0 != (spanContext.TraceIDHi == 0 && spanContext.TraceID == 0)) {
+		sensor.logger.Debug("only one of Instana ID headers present",
```

Comment:
This might be a personal preference, but shall we make it clearer that something is wrong by using words like "broken" or "corrupted"? Something like "broken instana trace context" and format ids there using `instana.Formap{ID,LongID}" since it's more familiar to the user.

Reply:
Makes sense what you suggested. The initial idea was to be very specific about the issue, but from another side, it was not clear that something is wrong 😃